src/types.d.ts

```diff
@@ -278,6 +278,8 @@ export const AudioWorkletProcessor: {
 };
 
 export interface WamEnv {
+    /** WAMs should put here under `moduleId` their dependencies. */
+    readonly dependencies: Record<string, any>;
```

Comment:
Overall I like the solution for giving each module it's own scope, nice job!

What I was saying in Slack was that I felt it might be cleaner to hide the implementation details from the caller, so instead of exposing how the module's scope is stored in `dependencies` and having to request that each WAM use `dependencies` properly, the WamEnv API should just have a method `getModuleScope(moduleId)` that hides the implementation details from the WAMs.  That would also allow it the implementation to change in the future in case there was additional work to do in WamEnv before returning the module's scope (like some sanity check or sandboxing or something)